integration/presto/src/main/prestosql/org/apache/carbondata/presto/CarbondataPageSource.java

```diff
@@ -356,7 +356,6 @@ private PrestoCarbonVectorizedRecordReader createReaderForColumnar(HiveSplit car
     QueryModel queryModel = createQueryModel(carbonSplit, tableHandle, columns, conf);
     if (isDirectVectorFill) {
       queryModel.setDirectVectorFill(true);
-      queryModel.setPreFetchData(false);
```

Comment:
@QiangCai : During streaming support for presto [#3001]
you made prefetch as false. 
Any particular reason or issue you found with that?

Reply:
No, we can enable it.

Reply:
ok. Thanks